### PR TITLE
Add Guess The Animal Solution

### PR DESCRIPTION
**Pull request by HarambeTheGiraffe**

### CODE REVIEW

**Comment by maggie-j-liu** [comment]
Please follow all the steps [here](https://usaco.guide/general/contributing?lang=cpp#adding-a-solution).
In particular, add the solution ID to the problem constructor, use tabs instead of spaces, and make sure each line is no more than 80 characters long.

File: solutions/USACO jan19-guess-the-animal.mdx [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+id: usaco-839
+source : USACO Bronze 2019 January
+title: Guess The Animal
+author: Ethan Wu
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_guess_bronze_jan19.html)
+<LanguageSection>
```
Comment (maggie-j-liu) [inline solutions/USACO jan19-guess-the-animal.mdx:9]:
```suggestion
<LanguageSection>

```

File: solutions/USACO jan19-guess-the-animal.mdx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+<JavaSection>
+
+## Implementation
```
Comment (maggie-j-liu) [inline solutions/USACO jan19-guess-the-animal.mdx:12]: Put the Implementation heading outside the LanguageSection and add the time complexity.

File: solutions/USACO jan19-guess-the-animal.mdx [hunk 3]
```diff
@@ -0,0 +1,73 @@
+---
+id: usaco-839
+source : USACO Bronze 2019 January
```
Comment (maggie-j-liu) [inline solutions/USACO jan19-guess-the-animal.mdx:3]:
```suggestion
source: USACO Bronze 2019 January
```

File: solutions/USACO jan19-guess-the-animal.mdx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_guess_bronze_jan19.html)
+<LanguageSection>
+<JavaSection>
+
+## Implementation
+
+```java
+
```
Comment (maggie-j-liu) [inline solutions/USACO jan19-guess-the-animal.mdx:15]:
```suggestion
```

File: solutions/USACO jan19-guess-the-animal.mdx [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+        /*
+        So far, Elsie has narrowed down her guesses to a couple of animals. In order to find the specific animal Bessie
+        is thinking of, she needs ask for one more trait that will be distinctive to the specific animal. So as a 
+        result we increment the answer by one. 
+        */
+
+
+        out.close();
+    }
+
```
Comment (maggie-j-liu) [inline solutions/USACO jan19-guess-the-animal.mdx:65]:
```suggestion
```

File: solutions/USACO jan19-guess-the-animal.mdx [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+                        The more qualities an animal shares with all the other animals, the more
+                        yes's are needed in order to find this animal. Since you trying to find the max
+                        amount of yes's, you simply find the animal who shares the most amount of similarities
+                        */
+                    }
+                }
+                ans = Math.max(ans, counter);
+                //compare the amount of similarities with the largest amount of similarities found so far
+            }
+
```
Comment (maggie-j-liu) [inline solutions/USACO jan19-guess-the-animal.mdx:53]:
```suggestion
```

File: solutions/USACO jan19-guess-the-animal.mdx [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+        }
+        out.println(ans + 1);
+        /*
+        So far, Elsie has narrowed down her guesses to a couple of animals. In order to find the specific animal Bessie
+        is thinking of, she needs ask for one more trait that will be distinctive to the specific animal. So as a 
+        result we increment the answer by one. 
+        */
+
+
```
Comment (maggie-j-liu) [inline solutions/USACO jan19-guess-the-animal.mdx:62]:
```suggestion
```

File: solutions/USACO jan19-guess-the-animal.mdx [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+        out.close();
+    }
+
+}
+
```
Comment (maggie-j-liu) [inline solutions/USACO jan19-guess-the-animal.mdx:67]:
```suggestion
```

**Review by maggie-j-liu — CHANGES_REQUESTED** [review]

File: solutions/USACO jan19-guess-the-animal.mdx (hunk 2 above)
Comment (jessechoe10) [reply solutions/USACO jan19-guess-the-animal.mdx:13]: Err... I don't think we need the time complexity for bronze problems... Putting time complexity is optional.

Comment (maggie-j-liu) [reply solutions/USACO jan19-guess-the-animal.mdx:13]: Oh, sorry, I didn't know that. @HarambeTheGiraffe, add the time complexity if you want, then.

**Comment by maggie-j-liu** [comment]
> Please follow all the steps [here](https://usaco.guide/general/contributing?lang=cpp#adding-a-solution).
> In particular, add the solution ID to the problem constructor, use tabs instead of spaces, and make sure each line is no more than 80 characters long.

@HarambeTheGiraffe

**Comment by jessechoe10** [comment]
@HarambeTheGiraffe pls try to address @maggie-j-liu 's comment :stuck_out_tongue: . I will close this PR if you don't change.

**Pull request closed without merging**